.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "OpenAssetIO"]
+	path = OpenAssetIO
+	url = https://github.com/OpenAssetIO/OpenAssetIO.git
```

Comment:
Nit
Just an observation, just looking in the editor config, indent_style is space.

However, this looks like tab indentation (going by the size, and the granularity of what I can select)

Reply:
Ah just realised, this is an autogenerated file, so I think OK to leave it as it is.

Reply:
Autogenerated by what?
I thought I've had to craft these by hand before.

Reply:
`git submodule add` IIRC

Reply:
TIL that there are tabs in other .gitmodules I've just looked at in the ecosystem! So... moot point. Nothing to see here.